examples/gsm8k_tuning/math_agent.py

```diff
@@ -190,7 +191,7 @@ def solve_task(agent: Agent, env: Environment, task: dict, tape_file: str = "")
 
 def extract_result_value(sample) -> dict:
     sample = dict(sample)
-    expected_result = str(sample["answer"]).rsplit("####", maxsplit=1)[-1].strip().replace(",", ".")
+    expected_result = str(sample["answer"]).rsplit("####", maxsplit=1)[-1].strip().replace(",", "")
```

Comment:
It should break floats written as string, no?

Reply:
Hi @ollmer , just saw this comment now. Can we chat about this in Monday? With the previous implementation, I was getting the following error: sample["answer"] = "1,000,000", was getting converted to "1.000.000" which could not be parsed as a int.

Reply:
Sure. Also, let's check the dataset for decimal separator consistency. Maybe they're a bit different between samples